content/en/policy-as-code.md

```diff
@@ -8,20 +8,16 @@ draft: true
 
 ## What it is
 
-Policy as Code is the practice of storing the definition of policies as one or more files in machine readable and processable form. 
-This replaces the traditional model where policies are documented in human-readable form, in separate documents.
+Policy as Code is the practice of storing the definition of policies as one or more files in machine-readable and processable form. 
+This replaces the traditional model where policies are documented in human-readable form in separate documents.
 
 ## Problem it addresses
 
-Building applications and infrastructures is often constrained by a lot of policies, that are defined by an organization, 
-e.g. security policies that forbid to store secrets in source code, 
-to run a container with superuser permissions, or to store some data outside a specific geo region. 
-It is very labor-intensive and error-prone for developers and reviewers to 
-manually check applications and infrastructure against documented policies. 
-Manual checking against policies cannot meet the responsiveness and scale requirements of cloud native applications.
+Building applications and infrastructures are often constrained by many policies that an organization defines, e.g., security policies that forbid storing secrets in source code, running a container with superuser permissions, or storing some data outside a specific geo region.
```

Comment:
```suggestion
Building applications and infrastructures are often constrained by many policies that an organization defines, 
e.g. security policies that forbid storing secrets in source code, running a container with superuser permissions, 
or storing some data outside a specific geo region.
```

Reply:
Hmm, you are splitting one sentence into three lines. I thought we wanted to keep one line per sentence (even if it is long) because, otherwise, editing a sentence might change its structure and affect the other lines.

Reply:
TBH I don't know about the agreement of keeping sentences in one line even if they are long. That is just my understanding of semantic line breaks. Also, for me personal it is easier to edit smaller lines. But I have no strong opinion here as long as we have a consensus in the team.

Reply:
As Noah proposed, it would be great to wrap long lines. 😊
What I wanted to suggest before was
- to wrap long lines in markdown codes (because it gives some benefits in editing, reviewing, reflecting, translating, ...),
- and when doing so, to use semantic line breaks, instead of routine line wrapping (e.g., every 80 characters)